### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LibbyMClark**
This looks great Mathieu! I just have a few places [in the brackets] where I think you could use a little more explanation/ transitional text. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,16 +80,16 @@ Ran 2 tests in 0.009s
 OK
 ```
 
-You've just ran your first tests locally and successfully! Let's see how you can have this task runned automatically for you everytime a new commit is done to your code repository.
+You've just run your first tests locally and successfully! Let's see how you can have this task run automatically for you everytime a new commit is done to your code repository.
 
 ## Using travis continuous integration plateform to perform test automatically
 
-In the second part of the tutorial you are going to use the SaaS version of travis-ci, one of the most popular continuous integration platform out there. Of course what you are going to do would apply perfectly to your own installtation of Travis-ci.
+In the second part of the tutorial you are going to use the SaaS version of travis-ci, one of the most popular continuous integration platforms out there. Of course what you are going to do would apply perfectly to your own installation of Travis-ci.
```
Comment (MBuffenoir) [inline README.md:87]: Sure about the s on platform here ? (for once I did not write it the french way ...)

File: README.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
-I will follow the spirit of the other tutorials I wrote and keep using plateforms, like Docker, that offers free plans for public projects.
+I will follow the spirit of the other tutorials I wrote and keep using platforms, like Docker, that offer free plans for public projects.
 
-Please register an account on the [Travis](https://travis-ci.org/) testing platform, preferably using your github account so that your repository can be automatically linked into your Travis account.
-
-## Create and run unit tests locally with docker
+Please register an account on the [Travis](https://travis-ci.org/) testing platform, preferably using your GitHub account so that your repository can be automatically linked into your Travis account.
 
+## Create and run unit tests locally with Docker
+[Can you add a sentence or two here about why you're doing this locally and on Docker first? ie Before I show you how to set up automatic testing, it's helpful to run a local test to get a good sense of how the process works and to make sure you're setting it up to get the outcome you want.]
```
Comment (MBuffenoir) [inline README.md:16]: Makes sense ...

File: README.md (hunk 1 above)
Comment (LibbyMClark) [inline README.md:87]: Yes - read it as "one of many...platforms" and it makes sense :)

Comment (LibbyMClark) [inline README.md:87]: sorry, meant "one of the most popular...platforms"

**Pull request merged**